src/openms/include/OpenMS/METADATA/ProteinHit.h

```diff
@@ -171,6 +196,11 @@ namespace OpenMS
     /// sets the coverage (in percent) of the protein hit based upon matched peptides
     void setCoverage(const double coverage);
 
+    /// returns the set of modified protein positions
+    const std::set<std::pair<Size, ResidueModification> >& getModifications() const;
+
+    /// sets the set of modified protein positions
+    void setModifications(std::set<std::pair<Size, ResidueModification> >& mods);
```

Comment:
you will also have to change the get/set Modifications fxns otherwise it probably wont compile